sstmac/hardware/nic/nic.cc

```diff
@@ -106,10 +106,10 @@ NIC::NIC(uint32_t id, SST::Params& params, Node* parent) :
   top_ = Topology::staticTopology(params);
 
   std::string subname = sprockit::sprintf("NIC.%d", my_addr_);
-  auto* spy = parent->registerMultiStatistic<int,uint64_t>(params, "spy_bytes", subname);
+  auto* spy = registerMultiStatistic<int,uint64_t>(params, "spy_bytes", subname);
   spy_bytes_ = dynamic_cast<StatSpyplot<int,uint64_t>*>(spy);
```

Comment:
If you know the actual type of spy, can you not just static_cast here?

Reply:
Added comments